experimenter/experimenter/settings.py

```diff
@@ -346,6 +346,15 @@
 REDIS_PORT = config("REDIS_PORT")
 REDIS_DB = config("REDIS_DB")
 
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"redis://{REDIS_HOST}:{REDIS_PORT}/{REDIS_DB}",
+        "TIMEOUT": None,
+    },
+}
```

Comment:
Oh I meant the other way around, like 

```py
CACHES = {
    "population_sizing": {
        ...
    }
}
```

and have no default that way 1) anybody trying to use the default will error out 2) people have to explicitly request the cache key they want to use

Reply:
Yea, tried to explain why I went with this in the other reply, but the important thing is that `default` is required, so our options are to either define `required` as something we won't use and `sizing` (or whatever) as redis, or just define `default` as redis.

I'll leave it as-is for now and we can modify as needed.

Reply:
One thing I just noticed is that django provides a [DummyCache](https://docs.djangoproject.com/en/3.2/topics/cache/#dummy-caching-for-development), so maybe this would be worth looking into as an option for `default`. I think it might be better to use something that, instead of silently succeeding without caching like DummyCache, explicitly errors if you try to use it so that anyone who tries to use it knows it won't work.

Not a big concern for the moment, but let me know what you think.